### DISCUSSION TITLE
en/latest/retries/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#12806] [General]
# en/latest/retries/

https://argo-workflows.readthedocs.io/en/latest/retries/

**Comment by lucastheisen** [comment]
The example for _Back-Off_ still defines `limit`.  Isn't limit and backoff mutually exclusive?  They are different ways of specifying the maximum number of attempts (with backoff its mathematical limit of attempts that fit into max time) right?

**Comment by cosmincatalin** [comment]
Can retry strategies be configured for individual steps?

**Reply by agilgur5** [reply]
Yes, that's the primary use case as far as I know. They are defined at the template-level. See also the [Field Reference](https://argo-workflows.readthedocs.io/en/latest/fields/#template)
